qa/tasks/cephfs/filesystem.py

```diff
@@ -255,7 +255,14 @@ def json_asok(self, command, service_type, service_id, timeout=None):
         proc = self.mon_manager.admin_socket(service_type, service_id, command, timeout=timeout)
         response_data = proc.stdout.getvalue().strip()
         if len(response_data) > 0:
-            j = json.loads(response_data)
+
+            def get_nonnumeric_values(value):
```

Comment:
any reason this isn't a general helper routine?

Reply:
It's very specific to this case. If we do want to this to make it a general helper, where do we place it so that it's avaiable to entire Ceph QA (since lots of Ceph QA use Python and can use JSON)?

Reply:
`qa/tasks/ceph_test_case.py`?

Reply:
specific to which mon command, btw?

Reply:
> specific to which mon command, btw?

Not to MON command but to the case where JSON printed includes "inf" or "-inf".

Reply:
ack.